internal/pkg/print/debug.go

```diff
@@ -57,6 +81,147 @@ func BuildDebugStrFromSlice(inputSlice []string) string {
 	return fmt.Sprintf("[%s]", sliceStr)
 }
 
+// buildHeaderMap converts a map to a user-friendly string representation.
+// This function also filters the headers based on the includeHeaders parameter.
+// If includeHeaders is empty, the default header filters are used.
+func buildHeaderMap(headers http.Header, includeHeaders []string) map[string]any {
```

Comment:
why return `map[string]any` if we are always constructing a `map[string]string`?

Reply:
The point is I'm trying to use the generic `BuildDebugStrFromMap` as much as possible, and that receives a map[string]any. So returning that type here makes it so that I can use the return to call that function

Reply:
But you can use `map[string]string` as an argument that is of type `map[string]any`

Reply:
I don't think so
![image](https://github.com/stackitcloud/stackit-cli/assets/33052643/2ed2ee6d-c207-42f6-a0fe-1f9d52fb818e)

Reply:
Unless I am missing something

Reply:
You're right, let's leave it like this